pkg/volume/iscsi/iscsi_util.go

```diff
@@ -394,6 +448,67 @@ func globalPDPathOperation(b iscsiDiskMounter) func(iscsiDiskMounter, string, *I
 	}
 }
 
+// Delete 1 block device of the form "sd*"
+func deleteDevice(deviceName string) error {
+	filename := fmt.Sprintf("/sys/block/%s/device/delete", deviceName)
+	fd, err := os.OpenFile(filename, os.O_WRONLY, 0)
+	if err != nil {
+		return err
```

Comment:
if the file doesn't exit, the device is already gone, don't error out here.

Reply:
Good point.